staking/target/idl/staking.json

```diff
@@ -1198,6 +1198,79 @@
           }
         }
       ]
+    },
+    {
```

Comment:
I think the idls are out of date and it's making CI fail

Reply:
Fixed